ArduPlane/quadplane.cpp

```diff
@@ -2177,6 +2243,23 @@ void QuadPlane::vtol_position_controller(void)
         } else {
             pos_control->set_xy_target(poscontrol.target.x, poscontrol.target.y);
         }
+
+#if PRECISION_LANDING == ENABLED
+        // run precision landing
```

Comment:
I'm not fully around pos_control, but it seems that when I override the it with ```pos_control->set_xy_target(target_pos.x, target_pos.y);``` there is less authority to move the craft to the target location - it can hover for many seconds approx 3m away from target coordinates horizontally, delaying the ```Q_POS_DESCEND``` transition